pages/notifications.vue

```diff
@@ -47,6 +47,8 @@ const filterIconMap: Record<mastodon.v1.NotificationType, string> = {
   'update': 'i-ri:edit-2-line',
   'admin.sign_up': 'i-ri:user-add-line',
   'admin.report': 'i-ri:flag-line',
+  'severed_relationships': 'i-ri:user-unfollow-line',
+  'moderation_warning': 'i-ri:error-warning-line',
```

Comment:
To satisfy type-checker, I chose these icons to add two new notification types:
 
![image](https://github.com/user-attachments/assets/941119fc-1731-4fa0-8136-b4c27f5e7324)
(last two icons are new)

Please refer to this documentation for explanations of each notification type: Notification - Mastodon documentation - https://docs.joinmastodon.org/entities/Notification/#type